<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="gridFrame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="4">
      <widget class="Line" name="line_e">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="Line" name="line_i">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="Line" name="line_j">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="Line" name="line_l">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="Line" name="line_h">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="Line" name="line_g">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="Line" name="line_d">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Line" name="line_f">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="Line" name="line_k">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_a">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="Line" name="line_c">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="Line" name="line_b">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="button_22">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>100</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="button_21">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>100</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="button_20">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>100</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="button_12">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>100</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="button_11">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>100</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="button_10">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>100</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="button_02">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>100</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="button_01">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>100</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button_00">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>100</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>line_e</zorder>
    <zorder>line_d</zorder>
    <zorder>line_c</zorder>
    <zorder>line_j</zorder>
    <zorder>line_i</zorder>
    <zorder>line_h</zorder>
    <zorder>line_l</zorder>
    <zorder>line_g</zorder>
    <zorder>line_f</zorder>
    <zorder>line_k</zorder>
    <zorder>line_b</zorder>
    <zorder>line_a</zorder>
    <zorder>button_22</zorder>
    <zorder>button_21</zorder>
    <zorder>button_20</zorder>
    <zorder>button_12</zorder>
    <zorder>button_11</zorder>
    <zorder>button_10</zorder>
    <zorder>button_02</zorder>
    <zorder>button_01</zorder>
    <zorder>button_00</zorder>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>water.png</pixmap>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>16</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Seleccione la casilla</string>
    </property>
   </widget>
   <widget class="QLabel" name="mensaje">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>400</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="jugador1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="x_image">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777212</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>x.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="jugador2">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="o_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>o.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="dificultad">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Facil</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dificil</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Elegir dificultad&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>gridFrame</zorder>
   <zorder>titulo</zorder>
   <zorder>mensaje</zorder>
   <zorder>jugador1</zorder>
   <zorder>x_image</zorder>
   <zorder>jugador2</zorder>
   <zorder>o_image</zorder>
   <zorder>dificultad</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
